eth/downloader/queue.go

```diff
@@ -62,15 +67,10 @@ type fetchResult struct {
 
 // queue represents hashes that are either need fetching or are being fetched
 type queue struct {
-	sync.Mutex
-
-	mode          SyncMode // Synchronisation mode to decide on the block parts to schedule for fetching
-	fastSyncPivot uint64   // Block number where the fast sync pivots into archive synchronisation mode
```

Comment:
Notice that both the `downloader` type `queue` have dropped their `fastSyncPivot` fields, shared state between downloader and state sync anymore. It's just passed around the required syncers and fetchers in the downloader.